cypress/integration/test_tables.js

```diff
@@ -1,136 +1,121 @@
 describe("Test Block Table", function () {
-    it("Click on Block Table view more button", function () {
-      cy.visit("http://localhost:3000/")
+  it("Click on Block Table view more button", function () {
+    cy.visit("http://localhost:3000/")
 
-      cy.wait(1000)
-    
-      cy.get("#block-table-more").click()
+    cy.wait(200)
 
-      cy.get('.data-table-header').should("not.contain", "Service")
+    cy.get("#block-table-more").click()
 
-    })
+    cy.get(".data-table-header").should("not.contain", "Service")
   })
+})
 
 describe("Test Txn Table", function () {
-    it("Click on Txn Table view more button", function () {
-      cy.visit("http://localhost:3000/")
+  it("Click on Txn Table view more button", function () {
+    cy.visit("http://localhost:3000/")
 
-      cy.wait(1000)
-    
-      cy.get("#txn-table-more").click()
+    cy.wait(200)
 
-      cy.get('.data-table-header').should("not.contain", "Num.")
+    cy.get("#txn-table-more").click()
 
-    })
+    cy.get(".data-table-header").should("not.contain", "Num.")
   })
+})
 
 describe("Test Block Table Pagination", function () {
-    it("Click on Block Table button pagination menu, assert results are same.", function () {
-      cy.visit("http://localhost:3000/")
+  it("Click on Block Table button pagination menu, assert results are same.", function () {
+    cy.visit("http://localhost:3000/")
+
+    cy.wait(200)
 
-      cy.wait(1000)
-    
-      cy.get("#block-table-more").click()
+    cy.get("#block-table-more").click()
 
     //   Get top hash
 
-    cy.get(".result-hash-cell").first().as('firstBlockHash')
+    cy.get(".result-hash-cell").first().as("firstBlockHash")
 
     cy.wait(100)
-        
-    cy.get("@firstBlockHash").then(($firstBlockHash) => { const value = $firstBlockHash.text()
 
-        cy.wait(500)
-        
-        cy.get('#pagination-next').click()
-        cy.get('#pagination-next').click()
-        cy.get('#pagination-next').click()
-        cy.get('#pagination-next').click()
-        cy.get('#pagination-next').click()
+    cy.get("@firstBlockHash").then(($firstBlockHash) => {
+      const value = $firstBlockHash.text()
 
+      cy.wait(500)
 
-        cy.get('#pagination-previous').click()
-        cy.get('#pagination-previous').click()
-        cy.get('#pagination-previous').click()
-        cy.get('#pagination-previous').click()
-        cy.get('#pagination-previous').click()
+      cy.get("#pagination-next").click()
```

Comment:
is there a way to make this more dry?
do for loops work here?

Reply:
I didn't really see something about how to do that with a loop. Rather, you can make it repeat the whole test over and over